PWGHF/HFC/TableProducer/correlatorDplusHadrons.cxx

```diff
@@ -324,7 +331,7 @@ struct HfCorrelatorDplusHadrons {
         if ((std::abs(particle2.pdgCode()) != 11) && (std::abs(particle2.pdgCode()) != 13) && (std::abs(particle2.pdgCode()) != 211) && (std::abs(particle2.pdgCode()) != 321) && (std::abs(particle2.pdgCode()) != 2212)) {
           continue;
         }
-        entryDplusHadronPair(getDeltaPhi(particle2.phi(), particle1.phi()),
+        entryDplusHadronPair(RecoDecay::constrainAngle(getDeltaPhi(particle2.phi(), particle1.phi()), -o2::constants::math::PI / 2.),
```

Comment:
Also here you can remove the RecoDecay::constrainAngle call, since it's already contained within the getDeltaPhi method

Reply:
Ciao Fabio,

Thanks for the comment, I fixed the comment.